File: 1_4/1task.py
print("Enter the number of characters")
N = 26
if N > 26:
    print("Incorrect data")
else:
    for i in range(N, 0, -1):
        for k in range(N, i-1, -1):
            Str = chr(ord('A') + k - 1)
            print(Str, end=" ")
        print()

File: 1_4/1task.cs
using System;
class Program {
    static void Main(string[] args) {
        Console.WriteLine("Enter the number of characters");
        int N = 26;
        if (N > 26) {
            Console.WriteLine("Incorrect data");
        }
        else {
            for (int i = N; i > 0; i--) {
                for (int k = N; k >= i; k--) {
                    char Str = (char)('A' + k - 1);
                    Console.Write(Str + " ");
                }
                Console.WriteLine();
            }
        }
    }
}